app/src/main/java/com/example/nigeriatelemedicineapp/api/services/RegisterPatientService.kt

```diff
@@ -1,13 +1,18 @@
 package com.example.nigeriatelemedicineapp.api.services
 
-import retrofit2.http.GET
+import com.example.nigeriatelemedicineapp.api.models.Patient
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.Headers
+import retrofit2.http.POST
 
 interface RegisterPatientService{
 
-
-
-
-
-
-
+    @Headers(
+        "accept: application/json",
+        "content-type: application/json"
+        //"authorization: Basic YWRtaW46QWRtaW4xMjM="
```

Comment:
Let's get rid of this too, unless there's a good justification for it

Reply:
@isears yup that was redundant